waterlog/src/reducers/AlertsReducer.js

```diff
@@ -0,0 +1,40 @@
+import {
+    FETCH_ALERTS_BEGIN,
+    FETCH_ALERTS_SUCCESS,
+    FETCH_ALERTS_FAILURE
```

Comment:
inconsistent with indentation used here `waterlog/src/Components/SegmentLeak.js`
choose which indentation you are going to use and stick with it throughout the project

Reply:
everything seems to be Ok on this, i dont see the inconsistency